libnmstate/ifaces/ifaces.py

```diff
@@ -198,6 +193,28 @@ def _apply_copy_mac_from(self):
             else:
                 iface.apply_copy_mac_from(port_iface.mac)
 
+    def _validate_copy_mac_from_iface_is_in_port(self, iface):
+        iface_port = iface.port
+        if iface.type == InterfaceType.OVS_INTERFACE:
+            ovs_bridges = [
```

Comment:
You could use lazy evaluation here.

Instead of list, construct a generator `ovs_bridges = ( .... )` and then use `if next(ovs_bridges):`. I think something like this should do it.

I will not fight for this, but it may be a little more efficient (even though I guess it won't save much in the whole scope of things, especially if all_kernel_ifaces.values() is not a generator itself).

Reply:
I think your suggestion reads better, apart from performance. Done